examples/notifications

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env node
+/*
+ * Copyright 2013-2018 The NATS Authors
```

Comment:
2019?

Reply:
Yes need to update all the touched ones.

Reply:
Fix all of them that you touched.